test_oscd.py

```diff
@@ -0,0 +1,9 @@
+from torchgeo.datasets import OSCD
```

Comment:
Thanks for testing this! However, we have a full test suite you can integrate with that will work better. See `tests/datasets` for examples of how we test the other datasets. Also, when you create a testing dataset in `tests/data`, make sure you don't include real images since that may violate a license agreement. Instead, create fake data using the docs [here](https://github.com/microsoft/torchgeo/tree/main/tests/data#readme). Let me know if you run into any problems!